battle.cpp

```diff
@@ -579,6 +614,11 @@ int Battle::Run( ARegion * region,
 	AddLine("");
 	AddLine(temp);
 	AddLine("");
+
+	if (!ass) {
```

Comment:
Do we want to add also assassination information in future too?

Reply:
As solution is extendable, then I would say yes